components/automate-cli/cmd/chef-automate/verify_test.go

```diff
@@ -14,710 +13,6 @@ const (
-					haOnpremDeploy:            false,
-					haOnPremAWSManagedDeploy:  false,
-					haOnPremCustManagedDeploy: true,
-					haAWSDeploy:               false,
-					haAWSManagedDeploy:        false,
-					standaloneDeploy:          false,
-					certificates:              false,
-				},
-			},
-			A2HARBFileExist: false,
-			mockFunc: &verification.VerificationMock{
-				VerifyOnPremCustManagedDeploymentFunc: func(configFile string) error {
-					return nil
-				},
-			},
-			ManagedServicesOn: false,
-			wantErr:           false,
-		},
-		{
-			name: "Test VerifyHaAWSDeploy",
-			args: args{
-				cmd:  &cobra.Command{},
-				args: []string{},
-				flagsObj: &verifyCmdFlags{
-					file:                      "",
-					haAWSProvision:            false,
-					haAWSManagedProvision:     false,
-					haOnpremDeploy:            false,
-					haOnPremAWSManagedDeploy:  false,
-					haOnPremCustManagedDeploy: false,
-					haAWSDeploy:               true,
-					haAWSManagedDeploy:        false,
-					standaloneDeploy:          false,
-					certificates:              false,
-				},
-			},
-			A2HARBFileExist: true,
-			mockFunc: &verification.VerificationMock{
-				VerifyHAAWSDeploymentFunc: func(configFile string) error {
-					return nil
-				},
-			},
-			ManagedServicesOn: false,
-			wantErr:           false,
-		},
-		{
-			name: "Test VerifyHaAWSManagedDeploy",
-			args: args{
-				cmd:  &cobra.Command{},
-				args: []string{},
-				flagsObj: &verifyCmdFlags{
-					file:                      "",
-					haAWSProvision:            false,
-					haAWSManagedProvision:     false,
-					haOnpremDeploy:            false,
-					haOnPremAWSManagedDeploy:  false,
-					haOnPremCustManagedDeploy: false,
-					haAWSDeploy:               false,
-					haAWSManagedDeploy:        true,
-					standaloneDeploy:          false,
-					certificates:              false,
-				},
-			},
-			A2HARBFileExist: true,
-			mockFunc: &verification.VerificationMock{
-				VerifyHAAWSManagedDeploymentFunc: func(configFile string) error {
-					return nil
-				},
-			},
-			ManagedServicesOn: true,
-			wantErr:           false,
-		},
-		{
-			name: "Test VerifyStandaloneDeploy",
-			args: args{
-				cmd:  &cobra.Command{},
-				args: []string{},
-				flagsObj: &verifyCmdFlags{
-					file:                      "",
-					haAWSProvision:            false,
-					haAWSManagedProvision:     false,
-					haOnpremDeploy:            false,
-					haOnPremAWSManagedDeploy:  false,
-					haOnPremCustManagedDeploy: false,
-					haAWSDeploy:               false,
-					haAWSManagedDeploy:        false,
-					standaloneDeploy:          true,
-					certificates:              false,
-				},
-			},
-			A2HARBFileExist: false,
-			mockFunc: &verification.VerificationMock{
-				VerifyStandaloneDeploymentFunc: func(configFile string) error {
-					return nil
-				},
-			},
-			ManagedServicesOn: false,
-			wantErr:           false,
-		},
-		{
-			name: "Test VerifyCertificates",
```

Comment:
Why are we removing this test, is the certificate flag not coming now ?

Reply:
I've added it.